spec/suites/layer/GeoJSONSpec.js

```diff
@@ -782,9 +762,10 @@ describe("L.GeoJSON functions", function () {
 	});
 
 	describe("#latLngsToCoords", function () {
-		it("returns a multidimensional array of coordinates", function () {
-			var coords = L.GeoJSON.latLngsToCoords([L.latLng(2, 1), L.latLng(4, 3)]);
-			expect(coords).to.eql([[1, 2], [3, 4]]);
+		it("returns a multidimensional array of coordinates (latitude, longitude, altitude)", function () {
+			var latLngs = [L.latLng(2, 1, 3), L.latLng(5, 4, 6)];
+			var coords = L.GeoJSON.latLngsToCoords(latLngs);
+			expect(coords).to.eql([[1, 2, 3], [4, 5, 6]]);
```

Comment:
In fact I'd prefer to keep both cases.
```js
		it("returns a multidimensional array of coordinates", function () {
			var lnglats = L.GeoJSON.latLngsToCoords([L.latLng(2, 1), L.latLng(4, 3)]);
			var lnglatalts = L.GeoJSON.latLngsToCoords([L.latLng(2, 1, 3), L.latLng(5, 4, 6)]);
			expect(lnglats).to.eql([[1, 2], [3, 4]]);
			expect(lnglatalts).to.eql([[1, 2, 3], [4, 5, 6]]);
```
or
```js
		it("returns a multidimensional array of coordinates", function () {
			var latlngs = [L.latLng(2, 1), L.latLng(4, 3)];
			var latlngalts = [L.latLng(2, 1, 3), L.latLng(5, 4, 6)];
			expect(L.GeoJSON.latLngsToCoords(latlngs)).to.eql([[1, 2], [3, 4]]);
			expect(L.GeoJSON.latLngsToCoords(latlngalts)).to.eql([[1, 2, 3], [4, 5, 6]]);
```
or
```js
		it("returns a multidimensional array of coordinates", function () {
			expect(L.GeoJSON.latLngsToCoords)
				.withArgs([L.latLng(2, 1), L.latLng(4, 3)])
				.to.eql([[1, 2], [3, 4]]);
			expect(L.GeoJSON.latLngsToCoords)
				.withArgs([L.latLng(2, 1, 3), L.latLng(5, 4, 6)])
				.to.eql([[1, 2, 3], [4, 5, 6]]);
```

Reply:
(and same for other cases)

Reply:
Ok will change that. Unfortunatly the more shorter form you proposed (which i would prefer)
```javascript
it("returns a multidimensional array of coordinates", function () {
			expect(L.GeoJSON.latLngsToCoords)
				.withArgs([L.latLng(2, 1), L.latLng(4, 3)])
				.to.eql([[1, 2], [3, 4]]);
			expect(L.GeoJSON.latLngsToCoords)
				.withArgs([L.latLng(2, 1, 3), L.latLng(5, 4, 6)])
				.to.eql([[1, 2, 3], [4, 5, 6]]);
```

 seems to only work for catching exceptions. It cannot test against a returned value.

Reply:
Indeed..